src/components/MultipleAvatars.js

```diff
@@ -1,7 +1,7 @@
 import React, {memo} from 'react';
 import PropTypes from 'prop-types';
 import {Image, Text, View} from 'react-native';
-import styles from '../styles/styles';
+import globalStyles from '../styles/styles';
```

Comment:
Not really into the `globalStyles` convention. Can we just keep this as `styles` and then if there are some more specific styles passed in as props give them a more descriptive name? Usually the prop styles have a specific purpose.